files/leaflet.esri.gp/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "leaflet.esri.gp",
+  "repo": "jgravois/esri-leaflet-gp",
+  "files": {
+    "basepath": ["dist/"]
```

Comment:
Add an `include` parameter to include only esri-leaflet-gp.js and esri-leaflet-gp.js.map. The rest are not needed